exec/cnex/cnex.c

```diff
@@ -172,6 +185,8 @@ int main(int argc, char* argv[])
                         "CallStack Height       : %" PRId32 "\n"
                         "Global Size            : %" PRIu32 "\n"
                         "Max Framesets          : %d\n"
+                        "Max Allocations        : %" PRIu64 "\n"
+                        "Remaining Objects      : %u\n"
```

Comment:
Why are these two values above different types? For the allocator that doesn't free until it's done, wouldn't they always be the same number?

Reply:
In this version, yes; they would.

Reply:
Changed All object counters to `size_t`.